src/framework/defaults.js

```diff
@@ -65,6 +67,30 @@ var Defaults = class Defaults {
     constructor(busName) {
         this._busName = busName;
         this._defaults = DEFAULTS[busName];
+        this._applyQuestOverridesSync();
+    }
+
+    _applyQuestOverridesSync() {
+        if (this._busName === 'com.endlessm.Hackdex_chapter_one') {
+            // Override for the "corrupted hackdex chapter 1" quest
+            const gameState = GameState.getDefault();
+            const key = 'app.com_endlessm_Hackdex_chapter_one.corruption';
+            let corruption;
+            try {
+                corruption = gameState.GetSync(key);
```

Comment:
curious: why not using `await gameState.Get(key)` here? Like you do  in the model.